.github/workflows/full.yml

```diff
@@ -459,9 +457,17 @@ jobs:
       - name: Add tag on version
         if: github.ref_type == 'tag'
         run: |
-          docker tag ${REGISTRY}/${CRANE_IMAGE_NAME}:${{ needs.changes.outputs.version }}
-          docker tag ${REGISTRY}/${DAGENT_IMAGE_NAME}:${{ needs.changes.outputs.version }}
-          docker tag ${REGISTRY}/${CLI_IMAGE_NAME}:${{ needs.changes.outputs.version }}
+          docker tag ${REGISTRY}/${CRANE_IMAGE_NAME}:${{ needs.changes.outputs.fallbacktag }} ${REGISTRY}/${CRANE_IMAGE_NAME}:${{ needs.changes.outputs.version }}
+          docker tag ${REGISTRY}/${DAGENT_IMAGE_NAME}:${{ needs.changes.outputs.fallbacktag }} ${REGISTRY}/${DAGENT_IMAGE_NAME}:${{ needs.changes.outputs.version }}
+          docker tag ${REGISTRY}/${CLI_IMAGE_NAME}:${{ needs.changes.outputs.fallbacktag }} ${REGISTRY}/${CLI_IMAGE_NAME}:${{ needs.changes.outputs.version }}
+          docker tag ${REGISTRY}/${CRANE_IMAGE_NAME}:${{ needs.changes.outputs.fallbacktag }} docker.io/dyrectorio/crane:${{ needs.changes.outputs.version }}
+          docker tag ${REGISTRY}/${DAGENT_IMAGE_NAME}:${{ needs.changes.outputsfallbacktag }} docker.io/dyrectorio/dagent:${{ needs.changes.outputs.version }}
```

Comment:
Could you use pre-defined variables for this `docker.io/dyrectorio`?

Reply:
Was planned, hence the draft status.